### PR TITLE
Remove old cloudfiles providers

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

File: providers/pom.xml [hunk 1]
```diff
@@ -52,9 +52,7 @@
     <module>softlayer</module>
     <module>hpcloud-compute</module>
     <module>hpcloud-blockstorage</module>
-    <module>hpcloud-objectstorage</module>
-    <module>cloudfiles-us</module>
-    <module>cloudfiles-uk</module>
+    <module>hpcloud-objectstorage</module>
```
Comment (andrewgaul) [inline providers/pom.xml]: Remove trailing whitespace from this line.

**Comment by andrewgaul** [comment]
:+1: Please remove legacy cloudfiles from karaf and cli before committing.

**Comment by zack-shoylev** [comment]
Karaf uses the cloudfiles providers in live tests, I think. Probably have to be replaced with the new providers and tested?

**Comment by andrewgaul** [comment]
Yeah it would be nice to update `CloudFilesUsLiveTest` to use rackspace-cloudfiles-us.

**Comment by zack-shoylev** [comment]
https://issues.apache.org/jira/browse/JCLOUDS-789

**Comment by zack-shoylev** [comment]
OK, I have the PRs for karaf and cli up as well.

**Comment by zack-shoylev** [comment]
merged

**Pull request closed without merging**